test/firtool/btor2-assertproperty.fir

```diff
@@ -0,0 +1,80 @@
+; RUN: firtool --btor2 %s | FileCheck %s
+
+FIRRTL version 3.2.0
+circuit Counter :
+  intmodule HasBeenResetIntrinsic :
```

Comment:
Consider intrinsic expressions instead of intmodule's, but this is fine as-is (even if in the future intmodule support is dropped, NBD).